### ISSUE TITLE
[Bettergram iOS] VoiceOver accessibility not working in Bettergram iOS

### ISSUE THREAD

**Issue by dw-b** [bettergram/bettergram#6]
(creating this here as as I cannot create issues against the iOS repo...)

Currently, the iOS app cannot be used by visually impaired people as it does not work with Apple's VoiceOver feature.  So, when launching the app, after the first few screens, blind people get the equivalent of a 'blank screen' (i.e. the screen reading software is unable to describe what's on the screen).

If you've rebuilt a new UI, adding support for Voiceover can be straight forward - you need to make sure accessibility labels are in place and so on:

https://developer.apple.com/documentation/uikit/accessibility/supporting_voiceover_in_your_app

https://medium.com/the-ministry-of-testing/adding-accessibility-identifiers-to-an-ios-app-c8e09d9c20f3

https://developer.apple.com/accessibility/ios/

Yours is one of the few, polished Telegram apps for iOS - appreciate your focus is on crypto but making the app accessible when the main Telegram applications _isn't_ would give you major publicity.

**Comment by ildar-gilmanov** [comment]
Please note that the development is stopped. Do not create new Issues.
